tests/conftest.py

```diff
@@ -555,3 +557,41 @@ def positions():
 @pytest.fixture
 def orientations(views, ups):
     return Orientations.from_view_up(views, ups)
+
+
+@pytest.fixture
+def sphericalvoronoi():
+    """ SphericalVoronoi object.
+    """
+    dihedral = 2 * np.arcsin(np.cos(np.pi / 3) / np.sin(np.pi / 5))
+    R = np.tan(np.pi / 3) * np.tan(dihedral / 2)
+    rho = np.cos(np.pi / 5) / np.sin(np.pi / 10)
+
+    theta1 = np.arccos(
+        (np.cos(np.pi / 5) / np.sin(np.pi / 5)) /
+        np.tan(np.pi / 3))
+
+    a2 = 2 * np.arccos(rho / R)
+
+    theta2 = theta1 + a2
+    theta3 = np.pi - theta2
+    theta4 = np.pi - theta1
+
+    phi1 = 0
+    phi2 = 2 * np.pi / 3
+    phi3 = 4 * np.pi / 3
+
+    theta = np.concatenate((
+        np.tile(theta1, 3),
+        np.tile(theta2, 3),
+        np.tile(theta3, 3),
+        np.tile(theta4, 3)))
+    phi = np.tile(np.array(
+            [phi1, phi2, phi3, phi1 + np.pi / 3,
+             phi2 + np.pi / 3, phi3 + np.pi / 3]), 2)
+    rad = np.ones(np.size(theta))
+
+    s = Coordinates(
+        phi, theta, rad,
+        domain='sph', convention='top_colat')
+    return SphericalVoronoi(s)
```

Comment:
What's the reason for these properties? (Compared to most of the other fixtures in this file, it's a little confusing that this is not a stub, but totally makes sense)

Reply:
This and the other new fixtures are no stubs because they are used to ensure that the encoding and decoding operates properly on Pyfar objects. Also, when class implementation changes in a way that would break the encoding/decoding the respective test should fail.

Reply:
I didn't see we are in the _feature/add-missing-copy_. Actually this belongs rather to feature/object-equality-check.

Reply:
It's called in test_copy, so it makes sense here... Maybe better readable implementation would be more suitable in this case, but the meaning of its properties will probably be more clear in another pull request

Reply:
Yes you are right. It is used by test_copy so nothing to fix here.